scripts/index.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var projects = [ ];
+
+function Project(obj) {
+  this.title = obj.title;
+  this.completedOn = obj.completedOn;
+  this.numFiles = obj.numFiles;
+  this.ghPages = obj.ghPages;
+  this.imageURL = obj.imageURL;
+  this.imgAlt = obj.imgAlt;
+  this.description = obj.description;
+}
+
+Project.prototype.renderProject = function() {
```

Comment:
Great work adding this method to your constructor in order to create an element and (eventually) append it to the DOM!